skbase/tests/conftest.py

```diff
@@ -178,6 +178,10 @@
 SKBASE_FUNCTIONS_BY_MODULE = SKBASE_PUBLIC_FUNCTIONS_BY_MODULE.copy()
 SKBASE_FUNCTIONS_BY_MODULE.update(
     {
+        "skbase.base._base": (
```

Comment:
I see you noticed this peculiarity ... I really regret the decision of using `skbase` itself for testing the lookup utilities, and not, say, a mock package in a subfolder.